OracleTuxedo/salt/samples/bankapp/stop_server.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# 
+# This script is used to attempt to gracefully shutdown the Tuxedo server before its pod is deleted.
+# It assumes that TUXDIR has been set
+
+export APPDIR=/u01/data/bankapp/
+LOG_FILE="${APPDIR}/stop_server.log"
+
+# shellcheck disable=SC1091
+source "${APPDIR}/bankvar.new"
+
+echo "stop server script kicked off at $(date)." > ${LOG_FILE}
+
+# Shutdown the domain
+tmshutdown -y
+
+touch "${SHUTDOWN_MARKER_FILE}"
```

Comment:
```suggestion
touch "$SHUTDOWN_MARKER_FILE"
```

This script doesn't seem to set `$SHUTDOWN_MARKER_FILE` either.

Reply:
Set in line 46 in start.sh and which is inherited by this stop script due to the `source "${APPDIR}/bankvar.new` at line 13 in this file.